portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -115,11 +130,43 @@ private User getUserFromRequest(HttpServletRequest request){
     }
 
     private Comment getCommentFromRequest(String senderId, HttpServletRequest request) {
-        String message = request.getParameter("comment");
-        Comment newComment = new Comment(senderId, message);
+        String message = request.getParameter(COMMENT_INPUT_NAME);
+        @Nullable String imgUrl = getUploadedFileUrl(request, IMG_INPUT_NAME);
```

Comment:
I don't think it is necessary to annotate local variable.

Reply:
removed annotation